Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/meals.yml

```diff
@@ -275,3 +275,134 @@
     tags:
     - Vegetable
     - Fruit
+  
+- type: entity
+  name: poached pear
```

Comment:
This should be "poached pears", for consistency

Reply:
Should be resolved.